vrt-examples/src/App.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+
+const componentsContext = require.context(
+  '../../src/components',
+  true,
+  /\.example\.js$/
+);
+const allComponents = componentsContext.keys().reduce((components, file) => {
+  const Comp = componentsContext(file);
+  const label = file.substring(file.lastIndexOf('/') + 1);
+  // eslint-disable-next-line no-param-reassign
+  components[label] = Comp;
+  return components;
+}, {});
+
+class App extends React.Component {
+  render() {
+    return (
+      <Router>
+        <Switch>
+          <Route path="/" exact component={() => <div>Home</div>} />
+          {Object.values(allComponents).map(Component => (
```

Comment:
create a route from all `example.js` files, using their exported `routePath` and `component`